yaas/config.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+
+# These default configs are overriden based on
+# command line arguments or environment variables
+
+server_url = None   # Must be overriden
+username = 'admin'
+password = 'admin'
+args = None
+
+def print_request_and_response(res, *args, **kwargs):
```

Comment:
Is this the same idea? http://stackoverflow.com/questions/20658572/python-requests-print-entire-http-request-raw
What is `*args, **kwargs` for?